src/test/java/org/fundacionjala/pivotal/runner/Runner.java

```diff
@@ -0,0 +1,21 @@
+package org.fundacionjala.pivotal.runner;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+import org.testng.annotations.DataProvider;
+
+/**
+ * Cucumber TestNG runner class.
+ */
+@CucumberOptions(
+        plugin = {"pretty"},
+        features = {"src/test/resources/features"},
+        glue = {"org.fundacionjala.pivotal"}
+)
+public final class Runner extends AbstractTestNGCucumberTests {
+    @Override
+    @DataProvider
+    public Object[][] scenarios() {
+        return super.scenarios();
+    }
+}
```

Comment:
For now please add the BeforeTest hook to set threadcount (provide the value of 1 as default for now):
```java
    @BeforeTest
    public void beforeAllScenarios() {
        System.setProperty("dataproviderthreadcount", "1");
    }
```

Reply:
It was added at last commit, thanks